pkg/clusteragent/admission/controllers/webhook/test_utils.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build kubeapiserver
+
+package webhook
```

Comment:
if this file is only used for the test you can name the package `webhook_test` it will only be seen from the test binary.

> Test files that declare a package with the suffix "_test" will be compiled as a
> separate package, and then linked and run with the main test binary.

Reply:
thanks for the tip! 💯  didn't know about it

Reply:
actually renaming the package `webhook_test` will require too many changes in the rest of go files in this package. I'll rename this file `utils_test.go` (with the `_test.go` suffix) to make sure it's only compiled in the test binary though.